[2024-10-03T17:11:30.447+0530] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-03T17:11:30.454+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:41:18.239286+00:00 [queued]>
[2024-10-03T17:11:30.460+0530] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:41:18.239286+00:00 [queued]>
[2024-10-03T17:11:30.460+0530] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-10-03T17:11:30.481+0530] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): validate_data> on 2024-10-03 11:41:18.239286+00:00
[2024-10-03T17:11:30.485+0530] {standard_task_runner.py:72} INFO - Started process 24603 to run task
[2024-10-03T17:11:30.488+0530] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'data_ingestion_dag', 'validate_data', 'manual__2024-10-03T11:41:18.239286+00:00', '--job-id', '57', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_dag.py', '--cfg-path', '/tmp/tmp9vx45i0h']
[2024-10-03T17:11:30.491+0530] {standard_task_runner.py:105} INFO - Job 57: Subtask validate_data
[2024-10-03T17:11:30.532+0530] {task_command.py:467} INFO - Running <TaskInstance: data_ingestion_dag.validate_data manual__2024-10-03T11:41:18.239286+00:00 [running]> on host Nithesh-Kumar.localdomain
[2024-10-03T17:11:30.609+0530] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='data_ingestion_dag' AIRFLOW_CTX_TASK_ID='validate_data' AIRFLOW_CTX_EXECUTION_DATE='2024-10-03T11:41:18.239286+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-03T11:41:18.239286+00:00'
[2024-10-03T17:11:30.610+0530] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-03T17:11:30.671+0530] {file_data_context.py:222} INFO - FileDataContext loading fluent config
[2024-10-03T17:11:30.673+0530] {config.py:187} INFO - Loading 'datasources' ->
[]
[2024-10-03T17:11:30.763+0530] {validator.py:1467} INFO - 	28 expectation(s) included in expectation_suite.
[2024-10-03T17:11:30.989+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T17:11:30.990+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   0%|          | 0/73 [00:00<?, ?it/s]
[2024-10-03T17:11:30.991+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:00, 2043.51it/s]
[2024-10-03T17:11:31.024+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   3%|2         | 2/73 [00:00<00:01, 58.79it/s]  
[2024-10-03T17:11:31.024+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:00, 86.73it/s]
[2024-10-03T17:11:31.055+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:   4%|4         | 3/73 [00:00<00:01, 45.83it/s]
[2024-10-03T17:11:31.061+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 238.09it/s]
[2024-10-03T17:11:31.087+0530] {logging_mixin.py:190} WARNING - Calculating Metrics:  23%|##3       | 17/73 [00:00<00:00, 174.13it/s]
[2024-10-03T17:11:31.114+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 588.50it/s]
[2024-10-03T17:11:31.114+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 588.50it/s]
[2024-10-03T17:11:31.124+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 588.50it/s]
[2024-10-03T17:11:31.125+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 588.50it/s]
[2024-10-03T17:11:31.125+0530] {logging_mixin.py:190} WARNING - Calculating Metrics: 100%|##########| 73/73 [00:00<00:00, 540.21it/s]
[2024-10-03T17:11:31.470+0530] {logging_mixin.py:190} INFO - Data quality check passed.
[2024-10-03T17:11:32.011+0530] {python.py:240} INFO - Done. Returned value was: None
[2024-10-03T17:11:32.016+0530] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-03T17:11:32.016+0530] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_ingestion_dag, task_id=validate_data, run_id=manual__2024-10-03T11:41:18.239286+00:00, execution_date=20241003T114118, start_date=20241003T114130, end_date=20241003T114132
[2024-10-03T17:11:32.068+0530] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-10-03T17:11:32.080+0530] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-10-03T17:11:32.095+0530] {local_task_job_runner.py:245} INFO - ::endgroup::
